create_infotexts.py

```diff
@@ -30,14 +30,15 @@ def load_places_mapping():
     places_df = places_df.set_index("Nyckelord")
     places_df.columns = ["freq", "commonscat", "wikidata"]
 
-    places_df.replace("-", np.nan)
+    places_df[places_df == "-"] = None
```

Comment:
It's the way pandas works: I use a [boolean filter](http://pandas.pydata.org/pandas-docs/stable/indexing.html#boolean-indexing) that says that each field in the entire DataFrame/data-dict where the value is '== "-"' should be set to None. I see by the results that it works.